HW04/Kate Fedotova/array_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rspec'
+require 'rubocop-rspec'
+require_relative 'arr.rb'
+
+# rubocop:disable Metrics/BlockLength
+describe Array do
+  describe 'my_each' do
+    let(:arr) { [1, 2, 3, 4] }
```

Comment:
You can use subject here, it's the same but will be more correct
```suggestion
    subject(:arr) { [1, 2, 3, 4] }
```